include/core/ib_particle.h

```diff
@@ -46,16 +46,16 @@ class IBParticle
     vx           = 2,
     vy           = 3,
     vz           = 4,
-    fx           = 5,
-    fy           = 6,
-    fz           = 7,
-    ox           = 8,
-    oy           = 9,
-    oz           = 10,
-    m            = 11,
-    tx           = 12,
-    ty           = 13,
-    tz           = 14,
+    ox           = 5,
+    oy           = 6,
+    oz           = 7,
+    fx           = 8,
+    fy           = 9,
+    fz           = 10,
+    tx           = 11,
+    ty           = 12,
+    tz           = 13,
+    m            = 14,
```

Comment:
so you switched the positions of the omega right? Are you sure the index was not used "hard coded" anywhere? Hopefully not.